projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -454,7 +454,7 @@ class='grunion-field-label{$type_class}" . ( $this->is_error() ? ' form-error' :
 				. $extra_attrs_string
 				. '>'
 				. esc_html( $label )
-				. ( $required ? '<span class="grunion-label-required">' . $required_field_text . '</span>' : '' )
+				. ( $required ? '<span class="grunion-label-required" aria-hidden="true">' . $required_field_text . '</span>' : '' )
```

Comment:
Screen readers say "required" for inputs with the `aria-required="true"` attribute. Here, we make the _Required_ label invisible to screen readers so that they don't read it twice.